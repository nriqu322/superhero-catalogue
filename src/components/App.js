import React from 'react';
import SearchForm from '../containers/searchForm';

function App() {
  return (
    <div className="App">
      Hello React!
      <div className="form-container">
        <SearchForm />
      </div>
    </div>

  );
}

export default App;
