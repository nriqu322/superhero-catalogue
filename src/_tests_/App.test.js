import React, { createElement } from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import App from '../components/App';
import Header from '../components/header';

it('renders app', () => {
  // const div = document.createElement('div');
  // ReactDOM.render(<App />, div);
});
