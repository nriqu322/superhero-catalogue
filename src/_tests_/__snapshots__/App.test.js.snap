// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app 1`] = `
<BrowserRouter>
  <Header />
  <Switch>
    <Route
      exact={true}
      path="/"
    >
      <Connect(SuperheroForm) />
      <Connect(SuperheroList) />
    </Route>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(SuperheroDetails)",
          "type": [Function],
        }
      }
      path="/details/:id"
    />
  </Switch>
</BrowserRouter>
`;
